src/test_workflow/benchmark_test/benchmark_args.py

```diff
@@ -131,3 +137,8 @@ def __init__(self) -> None:
         self.telemetry = args.telemetry
         self.telemetry_params = args.telemetry_params if args.telemetry_params else None
         self.logging_level = args.logging_level
+
+        if self.bundle_manifest is None and self.distribution_url is None:
+            raise Exception('Please provide either --bundle-manifest or --distribution-url to run the performance test.')
```

Comment:
What if both are not NONE, then which one will be taken?

Reply:
If you are talking about `--bundle-manifest` and `--distribution-version` being provided at the same time then workflow will default to `--bundle-manifest` and use that to run.